Bottom: eb2748402cd5024c83576b62490298f61c414326
Top:    eb2748402cd5024c83576b62490298f61c414326
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 22:56:59 +0000

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***







---


